### ISSUE TITLE
rundeck-cli-2.0.1-1.noarch.rpm is not signed

### ISSUE THREAD

**Issue by sumpfgottheit** [rundeck/rundeck-cli#462]
The rundeck-cli-2.0.1-1.noarch.rpm Package is not signed. The rundeck Repository ist set up to use the GPG Key with Keyid f192529298c4c654. If an unsigned RPM is encountered, at least Rhel refuses to install it.

Here the output of the 2.0.1 RPM - no Signature found: **Signature: (none)**
```
# rpm -qp --info rundeck-cli-2.0.1-1.noarch.rpm
Name        : rundeck-cli                  Relocations: /var/lib/rundeck-cli
Version     : 2.0.1                             Vendor: Rundeck, Inc.
Release     : 1                             Build Date: Tue 28 Jun 2022 07:50:50 PM CEST
Install Date: (not installed)               Build Host: fv-az290-443
Group       : System                        Source RPM: rundeck-cli-2.0.1-1-src.rpm
Size        : 7476203                          License: APL
Signature   : (none)
Packager    : runner
URL         : https://github.com/rundeck/rundeck-cli
Summary     : A commandline tool for Rundeck.
Description :
The rd program provides an all-in-one commandline tool to interact with Rundeck
```

Here the output of the correctly signed 2.0.0 RPM with **Signature: RSA/SHA1, Sat 21 May 2022 01:54:00 AM CEST, Key ID f192529298c4c654**
```
[root@osmirror ~]# rpm -qp --info rundeck-cli-2.0.0-1.noarch.rpm
warning: rundeck-cli-2.0.0-1.noarch.rpm: Header V4 RSA/SHA1 Signature, key ID 98c4c654: NOKEY
Name        : rundeck-cli                  Relocations: /var/lib/rundeck-cli
Version     : 2.0.0                             Vendor: Rundeck, Inc.
Release     : 1                             Build Date: Sat 21 May 2022 01:53:31 AM CEST
Install Date: (not installed)               Build Host: fv-az204-827
Group       : System                        Source RPM: rundeck-cli-2.0.0-1-src.rpm
Size        : 7476100                          License: APL
Signature   : RSA/SHA1, Sat 21 May 2022 01:54:00 AM CEST, Key ID f192529298c4c654
Packager    : runner
URL         : https://github.com/rundeck/rundeck-cli
Summary     : A commandline tool for Rundeck.
Description :
The rd program provides an all-in-one commandline tool to interact with Rundeck
```

Please sign the RPM and republish.

**Comment by otterz** [comment]
Hello, could you please sign the RPM and republish. Thanks!

**Comment by gschueler** [comment]
@sumpfgottheit @otterz published 2.0.2, please try that, it should work correctly

**Comment by otterz** [comment]
> @sumpfgottheit @otterz published 2.0.2, please try that, it should work correctly

Thx! :)

**Comment by sumpfgottheit** [comment]
Looks good! Thank you.
